x/concentrated-liquidity/lp_test.go

```diff
@@ -220,10 +220,10 @@ func (s *KeeperTestSuite) TestWithdrawPosition() {
 			// system under test parameters
 			// for withdrawing a position.
 			sutConfigOverwrite: &lpTest{
-				liquidityAmount: baseCase.liquidityAmount.QuoInt64(2),
+				liquidityAmount: baseCase.liquidityAmount.QuoRoundUp(sdk.NewDec(2)),
```

Comment:
Out of curiosity, what was the reason for this change? Just want to make sure we're being careful with updating rounding in tests

Reply:
I we do a normal division, the precision is barely off which prevents the test case of showing that half the liquidity withdrawn will withdraw 2500usd instead of 5000usd. With normal division, we leave a tiny bit of liq on the table which has us withdraw 2499.99 repeating. I could instead just call the liquidity amount directly instead of dividing